test/test_train_network.py

```diff
@@ -14,8 +15,8 @@
 class TestCore(object):
     def setup(self):
         # reset the tensorflow graph for new training
-        tf.reset_default_graph()
 
+        K.clear_session()
```

Comment:
If this line needs to be changed here, it likely needs to also be changed in the notebooks. In particular, verify that the three following notebooks are Keras-compatible: guide_dataset_building.ipynb, training_guideline.ipynb, and morphometrics_extraction.ipynb.

Reply:
I will look into these.

Reply:
I checked them and there was no need of it as all three notebooks are already keras compatible.  Just hard coding of epochs in training_guideline had to be fixed which is done in (ca26c63)